tests/10-proxy.sh

```diff
@@ -33,6 +33,12 @@ SVC_IP4="2.2.2.2"
 cleanup
 logs_clear
 
+function no_service_init {
```

Comment:
@ianvernon needs restructure in ginkgo

Reply:
@jrajahalme can you file a ticket with the `area/CI` label to make sure that these test changes make it into the Ginkgo framework ? Thanks.

Reply:
#2005